packages/event-bus/tests/unit/publish.test.ts

```diff
@@ -0,0 +1,17 @@
+import {EventEmitterEventBus} from "../../src";
+
+describe("Given Event Emitter", () => {
+    let eventEmitter: EventEmitterEventBus;
+
+    beforeAll(() => {
+        eventEmitter = new EventEmitterEventBus();
+    });
+
+    describe("When called to publish a message", () => {
+        it("Then it should publish the message", () => {
+            jest.spyOn(eventEmitter, 'publish');
+            eventEmitter.publish({name: "THIS_IS_MY_EVENT"});
+            expect(eventEmitter.publish).toHaveBeenCalledWith({name: "THIS_IS_MY_EVENT"});
+        })
```

Comment:
This test doesn't really test anything :p The pb is that the implementation is not using DI so it must be refactored before we re-write a good test for it
It can be commented for now